src/kili/entrypoints/base.py

```diff
@@ -0,0 +1,20 @@
+"""Base class for entrypoints dealing with GraphQL operations."""
+from typing import Optional, Type, TypeVar
+
+import requests
+
+from kili.core.graphql.graphql_client import GraphQLClient
+from kili.core.helpers import format_result
+
+T = TypeVar("T")
+
+
+class BaseOperationEntrypointMixin:
```

Comment:
`format_results` is used in each operation entrypoint, did I miss something?

Reply:
There are 3 format_results:
- the function itsel with the logic in the helper.py
- a new method of BaseOperationEntrypointMixin here
- a new method of the kili client

I just feel that adding only one new method is enough, why adding 2 of them ?
It would be easier to keep only the one in the kili client because the http_client and the graphql-client are defined there and that every entrypoint have access to the kili client. However is a function that is only related to graphql and not kili, so we can maybe keep the one in this file and delete the one in the kili client ?

Reply:
OK got it, I will remove the one from the kili client, and only keep the signature to keep the base class abstract. The one in the helper is used to keep the compatibility with the optional argument. I could have made it mandatory everywhere I admit.

Reply:
I finally did it the other way around, it is more easily testable